<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QMainWindow" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="FancyTabBar" name="sidebar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget_Pages"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="toolTip">
     <string>File</string>
    </property>
    <property name="statusTip">
     <string>File</string>
    </property>
    <property name="whatsThis">
     <string>File</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuService">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>Service</string>
    </property>
    <property name="statusTip">
     <string>Service</string>
    </property>
    <property name="whatsThis">
     <string>Service</string>
    </property>
    <property name="title">
     <string>Service</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>Help</string>
    </property>
    <property name="statusTip">
     <string>Help</string>
    </property>
    <property name="whatsThis">
     <string>Help</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <widget class="QMenu" name="menuToolbars">
    <property name="toolTip">
     <string>Toolbars</string>
    </property>
    <property name="statusTip">
     <string>Toolbars</string>
    </property>
    <property name="whatsThis">
     <string>Toolbars</string>
    </property>
    <property name="title">
     <string>Toolbars</string>
    </property>
    <addaction name="actionVisibleToolbarMain"/>
   </widget>
   <widget class="QMenu" name="menuNote">
    <property name="toolTip">
     <string>Note</string>
    </property>
    <property name="statusTip">
     <string>Note</string>
    </property>
    <property name="whatsThis">
     <string>Note</string>
    </property>
    <property name="title">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="toolTip">
     <string>Window</string>
    </property>
    <property name="statusTip">
     <string>Window</string>
    </property>
    <property name="whatsThis">
     <string>Window</string>
    </property>
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionFullScreen"/>
    <addaction name="actionTopOfAllWindows"/>
    <addaction name="actionShowSidebar"/>
    <addaction name="actionShowStatusBar"/>
   </widget>
   <widget class="QMenu" name="menuScripts">
    <property name="toolTip">
     <string>Scripts</string>
    </property>
    <property name="statusTip">
     <string>Scripts</string>
    </property>
    <property name="whatsThis">
     <string>Scripts</string>
    </property>
    <property name="title">
     <string>Scripts</string>
    </property>
    <addaction name="actionShowScriptsManager"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuToolbars"/>
   <addaction name="menuNote"/>
   <addaction name="menuService"/>
   <addaction name="menuScripts"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBarMain">
   <property name="windowTitle">
    <string>Main</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionOpenDict"/>
   <addaction name="actionCloseDict"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Manager.qrc">
     <normaloff>:/Manager/quit</normaloff>:/Manager/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit</string>
   </property>
   <property name="whatsThis">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Manager.qrc">
     <normaloff>:/Manager/about</normaloff>:/Manager/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
   <property name="whatsThis">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenDict">
   <property name="icon">
    <iconset resource="Manager.qrc">
     <normaloff>:/Manager/open_dict</normaloff>:/Manager/open_dict</iconset>
   </property>
   <property name="text">
    <string>Open Dictionary</string>
   </property>
   <property name="statusTip">
    <string>Open Dictionary</string>
   </property>
   <property name="whatsThis">
    <string>Open Dictionary</string>
   </property>
  </action>
  <action name="actionCloseDict">
   <property name="icon">
    <iconset resource="Manager.qrc">
     <normaloff>:/Manager/close_dict</normaloff>:/Manager/close_dict</iconset>
   </property>
   <property name="text">
    <string>Close Dictionary</string>
   </property>
   <property name="iconText">
    <string>Close Dictionary</string>
   </property>
   <property name="toolTip">
    <string>Close Dictionary</string>
   </property>
   <property name="statusTip">
    <string>Close Dictionary</string>
   </property>
   <property name="whatsThis">
    <string>Close Dictionary</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="statusTip">
    <string>Documentation</string>
   </property>
   <property name="whatsThis">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionShowSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Sidebar</string>
   </property>
   <property name="statusTip">
    <string>Show Sidebar</string>
   </property>
   <property name="whatsThis">
    <string>Show Sidebar</string>
   </property>
  </action>
  <action name="actionVisibleToolbarMain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Main</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="statusTip">
    <string>Full Screen</string>
   </property>
   <property name="whatsThis">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Manager.qrc">
     <normaloff>:/Manager/settings</normaloff>:/Manager/settings</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="statusTip">
    <string>Settings</string>
   </property>
   <property name="whatsThis">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionShowStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Status Bar</string>
   </property>
   <property name="statusTip">
    <string>Show Status Bar</string>
   </property>
   <property name="whatsThis">
    <string>Show Status Bar</string>
   </property>
  </action>
  <action name="actionShowScriptsManager">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ScriptModule/ScriptModule.qrc">
     <normaloff>:/fugue-icons/script-smiley</normaloff>:/fugue-icons/script-smiley</iconset>
   </property>
   <property name="text">
    <string>Show Scripts Manager</string>
   </property>
   <property name="statusTip">
    <string>Show Scripts Manager</string>
   </property>
   <property name="whatsThis">
    <string>Show Scripts Manager</string>
   </property>
  </action>
  <action name="actionTopOfAllWindows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On top of all windows</string>
   </property>
   <property name="statusTip">
    <string>On top of all windows</string>
   </property>
   <property name="whatsThis">
    <string>On top of all windows</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FancyTabBar</class>
   <extends>QWidget</extends>
   <header>fancytabbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Manager.qrc"/>
  <include location="../ScriptModule/ScriptModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>Manager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBarMain</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionVisibleToolbarMain</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVisibleToolbarMain</sender>
   <signal>triggered(bool)</signal>
   <receiver>toolBarMain</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
